<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1612</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>1281</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1271</width>
        <height>581</height>
       </rect>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>431</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>60</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>110</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>titre</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>150</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>description</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>210</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>duree</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>310</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>date_fin</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>250</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>date_debut</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="idFormation">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_titre">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>100</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_description">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>160</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ajouter_f">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>420</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modfi_f">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>420</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_duree">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>210</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_date_de_debut">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>260</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_date_de_fin">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>310</y>
           <width>110</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>350</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>id_formateur</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_formateur">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>360</y>
           <width>141</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QTableView" name="tableView_formations">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>140</y>
          <width>561</width>
          <height>271</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="supp_f">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1241</width>
        <height>671</height>
       </rect>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>431</width>
          <height>491</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>60</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="id_formateur">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_nom">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>110</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>150</y>
           <width>111</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>210</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Telephone</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>310</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Experience</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>250</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Specialite</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="idFormateur">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>60</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nom">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>100</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_mail">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>160</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ajouter_fr">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>420</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>ajouter</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modfi_fr">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>420</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_tel">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>210</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_specalite">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>250</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_exp">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>310</y>
           <width>111</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QTableView" name="tableView_formateur">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>130</y>
          <width>681</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="supp_fr">
        <property name="geometry">
         <rect>
          <x>1100</x>
          <y>510</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>90</y>
          <width>471</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="tri_fr">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>510</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>tri</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1612</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
